### PR TITLE
v1.6.0

### PR DESCRIPTION
**Pull request by ansis**
v1.6.0
and style spec v13.10.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## ✨ Features
 * Add ability to insert images into text labels using a format expression: `"text-field": ["format", "Some text", ["image", "my-image"], "some more text"]` ([#8904](https://github.com/mapbox/mapbox-gl-js/pull/8904))
```
Comment (chloekraw) [inline CHANGELOG.md]: using a `format` expression?

Comment (ansis) [reply CHANGELOG.md]: would "using an `image` expression within a `format` expression" be better?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
-* Add support for stretchable images (aka nine-patch images). Stretchable images can be used with `icon-text-fit` to draw resized images with unstretched corners and borders. ([#8997](https://github.com/mapbox/mapbox-gl-js/pull/8997))
+* Add support for stretchable images (aka nine-part or nine-patch images). Stretchable images can be used with `icon-text-fit` to draw resized images with unstretched corners and borders. ([#8997](https://github.com/mapbox/mapbox-gl-js/pull/8997))
 * Add "in" expression. It can check if a value is in an array (`["in", value, array]`) or a substring is in a string (`["in", substring, string]`) ([#8876](https://github.com/mapbox/mapbox-gl-js/pull/8876))
```
Comment (chloekraw) [inline CHANGELOG.md]: however we format the "in" expression should match "format" above

nit: Add "in" expression, which can check if...

File: CHANGELOG.md (hunk 1 above)
Comment (chloekraw) [reply CHANGELOG.md]: yep!

Comment (ansis) [reply CHANGELOG.md]: Or were you talking about type of quotes?

Comment (chloekraw) [reply CHANGELOG.md]: oh I was talking about the formatting. but I like your suggestion

**Pull request merged**